### PR TITLE
Add Galaxy LSP project

### PR DESCRIPTION
**Pull request by innovate-invent**
@bgruening I took the liberty of adding this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ A community project to get an overview about the Galaxy project
 - [usegalaxy.* tools](https://github.com/galaxyproject/usegalaxy-tools)
 - [managing tools with Ephemeris](https://github.com/galaxyproject/ephemeris)
 
+#### Tool development
+- [LSP server](https://github.com/galaxyproject/galaxy-language-server/) Microsoft language server protocol for Galaxy Tool XML. Also works with [Jetbrains products](https://plugins.jetbrains.com/plugin/10209-lsp-support)
```
Comment (bgruening) [inline README.md]: Have you tested it with Jetbrains?

**Comment by bgruening** [comment]
:kissing_heart: @innovate-invent

File: README.md (hunk 1 above)
Comment (beatrizserrano) [inline README.md]:
```suggestion
- [LSP server](https://github.com/galaxyproject/galaxy-language-server/) Microsoft Language Server Protocol for Galaxy Tool XML. Also works with [Jetbrains products](https://plugins.jetbrains.com/plugin/10209-lsp-support)
```

Comment (innovate-invent) [reply README.md]: At least that is what the plugin says it does

Comment (bgruening) [reply README.md]: Yeah, I was just wondering if you have tested it :)

Comment (innovate-invent) [reply README.md]: is there a running production instance of this server?

Comment (bgruening) [reply README.md]: No, I think you run those locally. Indeed VSC is starting the server for you.

**Pull request merged**